### ISSUE TITLE
Watson Q&A Service on Bluemix is Deprecated

### ISSUE THREAD

**Issue by JakePeyser** [IBM-Bluemix/IBM-Watson-Speech-QA-iOS#3]
The Watson Q&A service on Bluemix was deprecated on 11/19. Users can no longer follow along with the instructions. This should be updated to provide Q&A using the other Watson services available.

**Comment by triceam** [comment]
Yup, it's on the list.  I'm working with the Watson team to come up with a publicly accessible test Corpus using the new services.

**Comment by JakePeyser** [comment]
@triceam I suggest making this private for the time being until we can figure out a Q&A replacement

**Comment by JakePeyser** [comment]
Hey @triceam, can you either make this private or transfer ownership to your own account?
